extras/scripts/postwritehooks/apply_xslt.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Post-write hook script for MIK that validates MODS XML files.
+ * Works for single-file CONTENTdm and CSV import packages as well as
+ * newspaper issue packages, and can be extended to handle the MODS.xml
+ * files created by other MIK toolchains.
+ */
+
+require 'vendor/autoload.php';
+
+use Monolog\Logger;
+use Monolog\Handler\StreamHandler;
+use GuzzleHttp\Client;
```

Comment:
Sorry I didn't notice this earlier, but as far as I can tell you don't use Guzzle in this script. Can you remove this line?